src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/custom-timing-details.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Use $har for custom timing details
+tags:
+    - Synthetics
+    - Synthetic monitoring
+    - Scripting monitors
+    - Scripted API monitor
+metaDescription: You can now use $har to have your scripted API monitors report custom timing details that appear in the New Relic UI.
+redirects:
+---
+
+<Callout variant="important">
+You need [the new runtime](/docs/synthetics/synthetic-monitoring/getting-started/new-runtime) to retrieve new custom timing details from your scripted API monitors. 
+</Callout>
+
+$har is a library that gives you access to both [default node.js module](/docs/synthetics/synthetic-monitoring/scripting-monitors/import-nodejs-modules/#supported-core-modules) and [importable node.js modules](/docs/synthetics/synthetic-monitoring/using-monitors/manage-monitor-runtimes/#v0). These modules simplify:
+
+* Creating timers to test DNS and FTP requests that report to the New Relic UI.
+* Creating `SyntheticRequest` events for your scripted API monitors. 
+
+## Set up your custom timers
+
+Your timing details will appear in the New Relic UI as a waterfall view. To get those details, create your timing resources using `addResource()` or `getResource()`. You start these timers by using `startTimer()`. Likewise, you stop these timers by using `stopTimer()`. 
+
+To create a timing resource: 
+
+```
+const testTimer = $har.addResource('Test Timer');
+testTimer.startTimer();
+await sleep(100);
+testTimer.endTimer();
+```
+
+Alternatively, create a timer using `getResource()`:
+
+```
+$har.getResource('Test Timer').startTimer();
+await sleep(100);
+$har.getResource('Test Timer').endTimer();
+```
+
+## Retrieve specific timing details  
+
+Depending on your specific scripting needs, you can pull more granular timing details into the UI through a variety of attributes. These include: 
+
+* blocked
+* dns
+* connect
+* ssl
+* send
+* wait
+* receive
+
+These attributes appear with the `SyntheticRequest` event details. For example, to make a simple DNS attribute monitor:
```

Comment:
With the comment I mentioned above, we would already reference the benefits of the `SyntheticRequest` events. We could just change this to:

For example, to add timers to a simple DNS monitor: